src/Containers/Info/index.js

```diff
@@ -0,0 +1,378 @@
+import React from "react";
+import Seo from "../../Components/Seo";
+import ProjectColumns, { Column } from "../../Components/ProjectColumns";
+import { Link } from "react-router-dom";
+import {
+  container,
+  infoWrapper,
+  intoTitle,
+  infoContainer,
+  infoLink,
+  citeLink,
+  citeText,
+  nnTitle,
+} from "./style.module.scss";
+
+export default function Info() {
+  return (
+    <div className={container}>
+      <Seo />
+      <ProjectColumns className={infoWrapper}>
+        <Column>
+          <h2 className={intoTitle}>Curatorial Statement</h2>
+          <h3>May 20, 2020</h3>
```

Comment:
instead of making these `<h3>`, accessibility-wise these would be better placed in the same `<h2>` block using a `<span className="blah">` to style differently

Reply:
i'll make these changes